src/configs/agents/iOSChrome.jsx

```diff
@@ -0,0 +1,4 @@
+const iOSChrome = () => {
+    return navigator.userAgent.indexOf('CriOS') >= 0;
```

Comment:
Codacy has a fix for the issue: [Replace `··return·navigator.userAgent.indexOf('CriOS'` with `return·navigator.userAgent.indexOf("CriOS"`](https://app.codacy.com/gh/mohammadnedaei/Nearby-Locations-React/pullRequest?prid=9855911)
```suggestion
  return navigator.userAgent.indexOf("CriOS") >= 0;
```